app/helpers/content_helper.rb

```diff
@@ -73,4 +73,9 @@ def service_name
   def privacy_policy_url
     Course.config.privacy_policy_url
   end
+
+  # @return [Boolean]
+  def updated_content?
+    content.created_at.to_fs(:long) < content.updated_at.to_fs(:long)
```

Comment:
This is not a view layer concern, this is a model concern. We also don't need any formatting of timestamps to accomplish this.